patches/tModLoader/Terraria/ModLoader/DamageClass.cs

```diff
@@ -25,36 +23,72 @@ public abstract class DamageClass : ModType
 		/// </summary>
 		public ModTranslation ClassName { get; internal set; }
 
-		/// <summary> This is the name that will show up when an item tooltip displays 'X [ClassName]'. This should include the 'damage' part. </summary>
+		/// <summary>
+		/// This is the name that will show up when an item tooltip displays 'X [ClassName]'.
+		/// This should include the 'damage' part.
+		/// </summary>
 		public string DisplayName => DisplayNameInternal;
 
 		internal protected virtual string DisplayNameInternal => ClassName.GetTranslation(Language.ActiveCulture);
 
+		public StatInheritanceData DoubleCheckStatInheritanceData(DamageClass damageClass, Player player, Item item) {
```

Comment:
well, no, because `GetBaseModifierInheritance` and `GetDynamicModifierInheritance` aren't the same method

Reply:
Ah, I was saying they can have the same name, because they have different parameters, but as above I'm in favour of removing it. All tML hooks need sensible examples.